packages/now-routing-utils/src/index.ts

```diff
@@ -21,7 +21,14 @@ export { getCleanUrls } from './superstatic';
 export { mergeRoutes } from './merge';
 export { appendRoutesToPhase } from './append';
 
-const VALID_HANDLE_VALUES = ['filesystem', 'hit', 'miss'] as const;
+const VALID_HANDLE_VALUES = [
+  'filesystem',
+  'hit',
+  'miss',
+  'rewrite',
+  'error',
+] as const;
+const VALID_HANDLE_ERROR_KEYS = ['handle', 'src', 'dest', 'status'];
```

Comment:
Lets make this a Set

```suggestion
const validHandleErrorKeys = new Set(['handle', 'src', 'dest', 'status']);
```